styles.css

```diff
@@ -458,3 +456,278 @@ form {
 .message-label {
   padding-bottom: 40%;
 }
+
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  /* headliner page */
+  .toolbar {
+    padding-left: 10%;
+    width: 90vw;
+    margin-right: 0;
+  }
+
+  .toolbar-links {
+    color: #344563;
+    text-decoration: none;
+    display: flex;
+    justify-content: start;
+    width: 20vw;
+    gap: 4%;
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .toolbar-links > a {
+    padding: 0;
+    margin: 0;
+  }
+
+  #work {
+    height: 320vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- I recommend you use the grid for two-dimensional containers other than flex which is used for one-dimensional content.